### ISSUE TITLE
How to load local image?

### ISSUE THREAD

**Issue by KristinWang** [rii-mango/Papaya#115]
I was trying to load an local image 'sample_image.nii.gz', which is used by Papaya test. But the load failed with no error in the papaya display and the console.
I did some debug and found that the loading seems to be aborted when decompressing the image.  I have some screen shots for it. Could anyone have any thoughts about this????

**The usage code:**
`var params = [];
 params['images'] = ['../testData/sample_image.nii.gz'];`
`<div class="papaya" data-params="params"></div>`

**Papaya viewer stucked:** 
![image](https://user-images.githubusercontent.com/35942925/38302192-6426ef34-3834-11e8-84c8-852681a333dc.png)

**The debug result:**
![image](https://user-images.githubusercontent.com/35942925/38302150-478b7066-3834-11e8-96a3-2b2ff58de2a1.png)

**Comment by martinezmj** [comment]
I'm not able to recreate this issue.  Probably the real issue is upstream, such as it can't even read the file or the file is corrupted somehow.

Can you share more of your code?  Make sure you are running the page on a web server -- it cannot read the image file if you just open the html locally.  If this is what you're trying to do, run it locally (not on a web server), I can give you some ideas on how to make it work.

**Comment by KristinWang** [comment]
Thanks very much for replying to this. I did some more testing and found that I didn't set the image file url correctly. It seems that image file url must be relative to the file where papaya is loaded, rather than the file where papaya is started(container created)!. 
In my real case, I load the papaya in the "index.html" through a <script> tag. But I manually created the papaya container inside one of my react component, which is located inside a sub hierarchy folder. And I tried to load image to papaya in the react component by specifying the image url relatively to the file where papaya container is created. 
Now I correct the image url, it works fine now. Next step, I will need papaya to load image files located in a server (our file server), hopefully I can make it work fine.
Sorry to bother you for my misunderstanding. By the way, I  really admire your work about creating papaya, maintaining it and open source to us.
Best regards!
